Rent.Service/Rent.Service.Application/Abstractions/ICancelRentRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Rent.Service.Application.Model;
+using Rent.Service.Application.Rents.Commands;
+using Rent.Service.Domain.Entity;
+
+namespace Rent.Service.Application.Abstractions;
+
+public interface ICancelRentRepository
```

Comment:
ICancelRentRepository? What is the reason to segregate interfaces in this case? Do you have more than one realization for this interfaces? Or is it too big and u don't need all of those methods from the interfaces? Read about interface segregation and YAGNI(if u did it for the future)